pkg/mimirtool/commands/alerts.go

```diff
@@ -109,9 +111,33 @@ func (a *AlertmanagerCommand) getConfig(_ *kingpin.ParseContext) error {
 		return err
 	}
 
-	p := printer.New(a.DisableColor)
+	if a.OutputDir == "" {
+		p := printer.New(a.DisableColor)
+		return p.PrintAlertmanagerConfig(cfg, templates)
+	}
+	return a.outputAlertManagerConfigTemplates(cfg, templates)
+}
 
-	return p.PrintAlertmanagerConfig(cfg, templates)
+func (a *AlertmanagerCommand) outputAlertManagerConfigTemplates(config string, templates map[string]string) error {
+	var baseDir string
+	baseDir, err := filepath.Abs(a.OutputDir)
+	if err != nil {
+		return err
+	}
+	log.Debug(fmt.Sprintf("writing the config file to %s", filepath.Join(baseDir, "config.yaml")))
+	err = os.WriteFile(filepath.Join(baseDir, "config.yaml"), []byte(config), os.FileMode(0o666))
+	if err != nil {
+		return err
+	}
+
+	for fn, template := range templates {
+		log.Debug(fmt.Sprintf("writing the template file %s", fn))
+		err = os.WriteFile(filepath.Join(baseDir, fn), []byte(template), os.FileMode(0o666))
```

Comment:
same comment about a variable for the path. Would also be nice to log the full path in the debug log